src/core/net/ip6.cpp

```diff
@@ -1135,6 +1148,10 @@ Error Ip6::SendRaw(Message &aMessage, bool aAllowLoopBackToHost)
     error = HandleDatagram(aMessage, aAllowLoopBackToHost ? kFromHostAllowLoopBack : kFromHostDisallowLoopBack);
     freed = true;
 
+#if OPENTHREAD_CONFIG_IP6_BR_COUNTERS_ENABLE
+    UpdateBorderRoutingCounters(header, messageLength, true);
```

Comment:
Same here. Can we use `aMessage.GetLength()` directly here (so to remove `messageLength` variable?)